docs/src/content/docs/utilities/Injectors/inject-local-storage.md

```diff
@@ -55,9 +55,8 @@ Options to configure the behavior of the local storage signal.
 Here's a basic example of using `injectLocalStorage`:
 
 ```typescript
-const username = injectLocalStorage<string>('username', {
-	defaultValue: 'Anonymous',
-	storageSync: true,
+const username = injectLocalStorage<string | undefined>('username', {
+	storageSync: true
 });
```

Comment:
I would move this below to showcase the storageSync method, instead of showcasing it as a basic example. 

A basic example would just be : `const username = injectLocalStorage<string>('username');` and tell the developer that it will also contain the null or undefined (whatever we use as default) value in the type